src/components/ProjectSettingsTabs/NotificationsTab/getFormSchema.ts

```diff
@@ -3,22 +3,18 @@ import { SpruceFormProps } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
 import widgets from "components/SpruceForm/Widgets";
 import { form } from "../utils";
-import { DefaultSubscriptionsField } from "./Fields/DefaultSubscriptionsField";
 import { FormState } from "./types";
 
 const { radioBoxOptions } = form;
 
 export const getFormSchema = (
-  useRepoSettings: boolean,
   repoData?: FormState
 ): {
   fields: Record<string, Field>;
   schema: SpruceFormProps["schema"];
   uiSchema: SpruceFormProps["uiSchema"];
 } => ({
-  fields: {
-    defaultSubscriptionsField: DefaultSubscriptionsField,
```

Comment:
can we maybe keep this here for now since I'll be working on that part of the page soon?

Reply:
Sorry -- removed because I thought we opted not to allow defaulting to repo for this section. Will re-add!

Reply:
Oh actually you're right, we did. But now worries, I'll do that cleanup soon.